ht3/ht3.go

```diff
@@ -0,0 +1,14 @@
+package ht3
+
+func average(array [6]int) float64 {
```

Comment:
Average function differs a bit from median. Please take a look at https://en.wikipedia.org/wiki/Median

Reply:
I can't understand what do you mean because I must to realize "Average" function

Reply:
yep, my fault. Missed the task.